### PR TITLE
after run in my host did some improvements.

### PR DESCRIPTION
**Pull request by armdev**
after run in my host did some improvements.

### CODE REVIEW

File: src/main/java/com/hendisantika/abcbank/AbcBankApplication.java [hunk 1]
```diff
@@ -6,9 +6,15 @@
...
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
 @SpringBootApplication
+@EnableJpaRepositories("com.hendisantika.abcbank.repository")
+@ComponentScan(basePackages = {"com.hendisantika"})
+@EntityScan("com.hendisantika.abcbank.entity")
```
Comment (hendisantika) [inline src/main/java/com/hendisantika/abcbank/AbcBankApplication.java:17]: It doesn't need since Spring Boot 1.x.x

Comment (hendisantika) [reply src/main/java/com/hendisantika/abcbank/AbcBankApplication.java:17]: Remove it

**Review by hendisantika — CHANGES_REQUESTED** [review]
Check my review

File: src/main/java/com/hendisantika/abcbank/AbcBankApplication.java (hunk 1 above)
Comment (armdev) [reply src/main/java/com/hendisantika/abcbank/AbcBankApplication.java]: Done

**Pull request merged**